### PR TITLE
improve popup and add description in new trip form expand icon

### LINKED ISSUE #205: When you click map markers, the popovers look bad
image
This looks terrible

### LINKED ISSUE #208: colloquial prompt opener slightly confusing
image

### PR DESCRIPTION
**Pull request by vee-16**
closes #205 & #208 

Screenshot 2023-08-06 at 1 05 44 AM
Screenshot 2023-08-06 at 1 06 11 AM

(icons may be redundant here, added it as sometimes the popup covers the whole icon. but design ideas encouraged)
Screenshot 2023-08-06 at 12 47 29 AM

### CODE REVIEW

**Comment by L0Lmaker** [comment]
Instead of a reference to GPT what do you guys think about having it something like: "Speak your mind 🧠"

File: client/src/components/TripElement/NewTripForm.js [hunk 1]
```diff
@@ -71,19 +71,23 @@ export function NewTripForm() {
...
+        onClick={() => setCompressed(!compressed)}
+      >
+        {compressed ? (
+          <>
+            <span className='relative'>Form</span>
+            <ExpandLessIcon fontSize='large' className='w-3' />
+          </>
+        ) : (
+          <>
+            <span className='relative'>Chat with GPT</span>
```
Comment (L0Lmaker) [inline client/src/components/TripElement/NewTripForm.js]:
```suggestion
            <span className='relative'>Speak Your Mind 🧠</span>
```

**Review by L0Lmaker — APPROVED** [review]
Code suggestion added

File: client/src/components/TripElement/NewTripForm.js (hunk 1 above)
Comment (vee-16) [reply client/src/components/TripElement/NewTripForm.js]: I like it

**Comment by Graywing13** [comment]
looks great! I especially like having the emoji in the popover. 

a small nit, is it possible to make the popovers exit x have a bit more padding? rn it's kinda close to the edge

**Comment by vee-16** [comment]
Screenshot 2023-08-06 at 11 12 47 PM

Updated popup

**Pull request merged**